Indexing/CodeTree.hpp

```diff
@@ -135,7 +135,7 @@ class CodeTree
     ILStruct* previous;
 
     unsigned isVarEqLit:1;
-    unsigned varEqLitSort:31;
```

Comment:
This changes the memory footprint from 32 bit (for compilers able to compile bitfields) to 1+64, which is probably at least 80 bits (on 64 bits platforms) due to alignment. 

In the future, for occasions where nothing but sorts' (not)identity is relevant for the use, we could instead term's id:

```
  /** For shared terms, this is a unique id used for deterministic comparison */
  unsigned _id;
```

Especially, after sorts get their own TermSharing instance, as planned, these will be small unsigneds uniquely numbering the sorts! (I.e. no need for `SortHelper::sortNum` as used, e.g., in FMB.)

Reply:
Sounds like the way to go!